sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -81,9 +81,11 @@
   "sideEffects": false,
   "autoPublish": false,
   "dependencies": {
+    "@azure/abort-controller": "^1.0.0",
     "@azure/core-asynciterator-polyfill": "^1.0.0",
     "@azure/core-auth": "^1.1.3",
-    "@azure/core-http": "^1.2.0",
+    "@azure/core-client": "1.0.0-beta.1",
```

Comment:
core-auth and core-tracing get pulled in by core-client too.

Reply:
I see but is there any advantage for pulling dependencies transitively instead of directly?

Reply:
less code? on the other hand it might be desirable to explicitly include them. I don't know which one is better, just that the old code didn't have them.